content/docs/guides/traefik-ingress.mdx

```diff
@@ -27,9 +27,11 @@ import Install from '../../examples/traefik-ingress/install.sh.md';
 
 :::danger
 
-Before You Proceed Since this guide was written, Pomerium has developed into a first-class proxy service, and provides an integrated [Ingress Controller](/docs/deploying/k8s/ingress) for Kubernetes.
+Starting v0.21.0, Pomerium will no longer support [**Forward Auth**](/docs/reference/forward-auth). Supporting Forward Auth requires Pomerium to route requests from third-party proxies to make access control decisions. This goes against zero-trust principles as specified in the BeyondCorp model, which states that all traffic should flow through a single proxy.
 
-This guide will remain available for those already using Traefik with Pomerium, but we strongly encourage new users to configure Pomerium as the single method of authentication and proxying.
+Since this guide was written, Pomerium has developed into a first-class proxy service, and provides an integrated [Ingress Controller](/docs/deploying/k8s/ingress) for Kubernetes.
+
+For previous versions of Pomerium up to v0.20.0, this guide will remain available for those already using Nginx with Pomerium, but we strongly encourage new users to configure Pomerium as the single method of authentication and proxying.
```

Comment:
the wording assumes we intend to delete this page and make a redirect to `https://0-19-0.docs.pomerium.com/docs/guides/traefik-ingress.mdx` but the page itself is not deleted.